envs/monkey_zoo/packer/files/chrome-creds.ps1

```diff
@@ -0,0 +1,339 @@
+			$sql += " VALUES (@origin_url, @action_url, @username_element, @username_value, @password_element, @password_value, @submit_element, @signon_realm, @date_created, @blacklisted_by_user, @scheme, @password_type, @times_used, @form_data, @display_name, @icon_url, @federation_url, @skip_zero_click, @generation_upload_status, @possible_username_pairs, @date_last_used, @moving_blocked_for, @date_password_modified);"
+
+			$password_value = EncryptPassword $password_value
+
+			$param = $cmd.Parameters.AddWithValue("@origin_url", $origin_url)
+			$param = $cmd.Parameters.AddWithValue("@action_url", $action_url)
+			$param = $cmd.Parameters.AddWithValue("@username_element", $username_element)
+			$param = $cmd.Parameters.AddWithValue("@username_value", $username_value)
+			$param = $cmd.Parameters.AddWithValue("@password_element", $password_element)
+			$param = $cmd.Parameters.AddWithValue("@password_value", $password_value)
+			$param = $cmd.Parameters.AddWithValue("@submit_element", $submit_element)
+			$param = $cmd.Parameters.AddWithValue("@signon_realm", $signon_realm)
+			$param = $cmd.Parameters.AddWithValue("@date_created", ([DateTimeOffset]::UtcNow.ToUnixTimeMilliseconds() * 1000))
+			$param = $cmd.Parameters.AddWithValue("@blacklisted_by_user", $blacklisted_by_user)
+			$param = $cmd.Parameters.AddWithValue("@scheme", $scheme)
+			$param = $cmd.Parameters.AddWithValue("@password_type", $password_type)
+			$param = $cmd.Parameters.AddWithValue("@times_used", $times_userd)
+			$param = $cmd.Parameters.AddWithValue("@form_data", (HexToByteArray($form_data)))
+			$param = $cmd.Parameters.AddWithValue("@display_name", $display_name)
+			$param = $cmd.Parameters.AddWithValue("@icon_url", $icon_url)
+			$param = $cmd.Parameters.AddWithValue("@federation_url", $federation_url)
+			$param = $cmd.Parameters.AddWithValue("@skip_zero_click", $skip_zero_click)
+			$param = $cmd.Parameters.AddWithValue("@generation_upload_status", $generation_upload_status)
+			$param = $cmd.Parameters.AddWithValue("@possible_username_pairs", (HexToByteArray($possible_username_pairs)))
+			#$param = $cmd.Parameters.AddWithValue("@id", '9') # AUTO INCREMENT
+			$param = $cmd.Parameters.AddWithValue("@date_last_used",  ([DateTimeOffset]::UtcNow.ToUnixTimeMilliseconds() * 1000))
+			$param = $cmd.Parameters.AddWithValue("@moving_blocked_for", (HexToByteArray($moving_blocked_for)))
+
+			$param = $cmd.Parameters.AddWithValue("@date_password_modified", ([DateTimeOffset]::UtcNow.ToUnixTimeMilliseconds() * 1000))
+
+			$cmd.CommandText = $sql
+			$res = $cmd.ExecuteNonQuery()
+
+			$cmd.dispose()
+
+			Write-Host "...credential imported!"
+
+		}
+
+		"Delete" {
+			Write-Host "Deleting all credentials..."
+
+			$cmd = $conn.CreateCommand()
+
+			$sql = "DELETE FROM main.logins"
+			$cmd.CommandText = $sql
+			$res = $cmd.ExecuteNonQuery()
+
+			$cmd.dispose()
+
+			Write-Host "...all credentials deleted!"
+
+		}
+}
+
+#$conn.Close()
```

Comment:
?

Reply:
I've fixed this script to work for our case, don't want to spend time improving readability or whatnot